onadata/libs/utils/csv_builder.py

```diff
@@ -561,6 +561,7 @@ def export_to(self, path, dataview=None):
         columns_with_hxl = self.include_hxl and get_columns_with_hxl(
             self.dd.survey_elements)
 
+        print columns
```

Comment:
✂️

Reply:
done in c091ca28abfbfa92583e2239b1bfaf9a65d2882f